### DISCUSSION TITLE
Format of the Inventory Data

### DISCUSSION THREAD

**Discussion by Manuela-RT** [NGEET/fates-tutorial#55] [Q&A]
I have the BCI inventory data but already in a format usable by ED 2.2. (see the attached image). Are these files also directly usable by FATES? If these files were produced having the whole 50ha plot, is it better to wait until Monday and do the processing with the inventory file you have in order to produce the files for FATES only with a single ha as recommended in the tutorial?

Thank you in advance.
![Screenshot 2024-09-06 143734](https://github.com/user-attachments/assets/a9171390-fff5-48c9-a250-7c44af0a26ea)

**Comment by JessicaNeedham** [comment]
Hi @Manuela-RT, great question. FATES and ED2 used to use the same inventory input files but the FATES files are now formatted slightly differently so I don't think your ED2 files will work. However, we've found a way of using the full 50 ha data set that doesn't take ages to load so we'll provide those files  for you on Monday. There's no need for you to process the BCI data - the files will be in the docker container already.
